src/components/theme/GoodsItemList/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import goodsItemList from '@data/goodsItemList';
+import { GoodsItem, Grid, Inner } from '@components/common';
+import { BREAK_POINTS } from '@assets/styles/variants';
+
+const GRID_GAP = 14;
+const GRID_COLUMNS = 4;
+
+export default function GoodsItemList() {
+  return (
+    <GoodsItemListContainer>
+      <Inner maxWidth={BREAK_POINTS.md}>
+        <Grid gap={GRID_GAP} columns={GRID_COLUMNS}>
+          {goodsItemList.map(({ id, imageSrc, subtitle, title, amount, target, wish }) => (
+            <GoodsItem key={id} {...{ imageSrc, subtitle, title, amount, target, wish }} />
+          ))}
```

Comment:
```suggestion
          {goodsItemList.map(({ id, imageSrc, subtitle, title, amount }) => (
            <GoodsItem key={id} imageSrc={imageSrc} subtitle={subtitle} title={title} amount={amount} />
          ))}
```

target, wish는 GoosItem 컴포넌트의 props로 선언하지 않은 속성인데 spread operator를 통해 전달해주고 있기 때문에 존재하지 않는 props라는걸 typescript가 잡아주질 못하고 있어요. 이런 경우 props key, value 매칭시켜서 직접 전달해주는게 좋아요.